ERCS/erc-social_sbt.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: draft
+title: Social SBT
+description: Social SBT, A Soul-Bound Token Standard for DAO Governance
+author: Ali Bertay SOLAK (@alibertay)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-12-27
+---
+
+## Abstract
+SocialSBT is a non-transferable ERC-721-based token standard that integrates a dynamic social point system for DAO governance. It aims to mitigate whale attacks and promote equitable, community-driven decision-making.
```

Comment:
Please consider other ERCs using/defining SBTs to either use them to build on top of or look at collaborating:
* ERC-7787: Soulbound Degradable Governance
* ERC-5114: Soulbound Badge
* ERC-5192: Minimal Soulbound NFTs
* ERC-5484: Consensual Soulbound Tokens
* ERC-5516: Soulbound Multi-owner Tokens
* ERC-5633: Composable Soulbound NFT, EIP-1155 Extension
* ERC-5727: Semi-Fungible Soulbound Token
* ERC-6239: Semantic Soulbound Tokens
* ERC-7704: Soul Resonance Token